### PR TITLE
make navbar sticky and fix image descriptions

### PR DESCRIPTION
**Pull request by ghost**
makes navbar stick to the top of the page when you scroll down

also fixed image descriptions and added alt tooltips

### CODE REVIEW

**Comment by 5HT2** [comment]
uhhh, doesn't seem to work for me
ezgif com-video-to-gif

**Comment by ghost** [comment]
hmm I'll retest

File: docs/index.md [hunk 1]
```diff
@@ -41,21 +41,21 @@ See [forgehax](https://github.com/fr1kin/forgehax) for an equivalent. Some featu
 	
 ***
```
Comment (5HT2) [inline docs/index.md]: Can we remove the `***\n`

**Review by 5HT2 — CHANGES_REQUESTED** [review]

File: docs/index.md (hunk 1 above)
Comment (ghost) [reply docs/index.md]: just that one or all of them??

Comment (5HT2) [reply docs/index.md]: @edwr the highlighted one that's there

Comment (5HT2) [reply docs/index.md]: Oh and the bottom one in images too please

**Comment by ghost** [comment]
It should work now

**Review by 5HT2 — APPROVED** [review]

File: docs/index.md [hunk 2]
```diff
@@ -41,21 +41,21 @@ See [forgehax](https://github.com/fr1kin/forgehax) for an equivalent. Some featu
...
-<img src="https://raw.githubusercontent.com/S-B99/kamiblue/assets/assets/readme/shulkerChat.png" width="500"/>
+<img src="https://raw.githubusercontent.com/S-B99/kamiblue/assets/assets/readme/shulkerChat.png" width="500" alt="Shulker preview being used in chat"/>
 
-CrystalAura targeting
+<p>CrystalAura targeting</p>
 
-<img src="https://raw.githubusercontent.com/S-B99/kamiblue/assets/assets/readme/crystalAura.png" width="500"/>
+<img src="https://raw.githubusercontent.com/S-B99/kamiblue/assets/assets/readme/crystalAura.png" width="500" alt="CrystalAura targeting"/>
 
 ***
```
Comment (5HT2) [inline docs/index.md]: Ye I missed this one, remove this as well please

**Review by 5HT2 — CHANGES_REQUESTED** [review]

File: docs/index.md [hunk 3]
```diff
@@ -24,22 +24,17 @@ This is currently in active development. When issues are being closed is by mile
 <details>
 	<summary>< Click to view disclaimers ></summary>
 
-***
-
 This is by no means a finished project, nor is it a "cheat" or "hack" for anything, it is a *utility* mod.
 
 See [forgehax](https://github.com/fr1kin/forgehax) for an equivalent. Some features in KAMI may be based on those of forgehax, and KAMI / KAMI Blue have some features it doesn't. KAMI Blue won't be based off of other mods unless said otherwise.
 
-***
```
Comment (5HT2) [inline docs/index.md:30]: Can you remove one line here xd

it's double lined by accident now

otherwise looks good c:

**Review by 5HT2 — CHANGES_REQUESTED** [review]

**Comment by 5HT2** [comment]
Looks good uwu. If you want before I had to sleep you can convert the disclaimer section to html to fix #3. I'll merge this in a sec

**Pull request merged**